libs/encryption-sdk/build.gradle

```diff
@@ -25,7 +25,6 @@ dependencies {
 
   // Encryption
   implementation "com.amazonaws:aws-encryption-sdk-java:2.4.0"
-  implementation "org.bouncycastle:bcprov-jdk15to18:${versions.bouncycastle}"
```

Comment:
This dependency is not used by encryption SDK and should be only included by the components that need it, right now it breaks at least `security` plugin since it completely messes up the security policies.

Reply:
@vikasvb90  fyi